RELEASING/email_templates/vote_pmc.j2

```diff
@@ -39,7 +39,7 @@ https://github.com/apache/{{ project_module }}/blob/{{ version_rc }}/UPDATING.md
 Public keys are available at:
 https://www.apache.org/dist/{{ project_module }}/KEYS
 
-The vote will be open for at least 72 hours or until the necessary number
+The vote will be open for at least 72 hours and until the necessary number
```

Comment:
Maybe "The vote will be left open until at least 72 hours have passed and the necessary number of votes (3) have been reached."